attributes.go

```diff
@@ -0,0 +1,256 @@
+package crypto11
+
+import (
+	"fmt"
+	"github.com/miekg/pkcs11"
+)
+
+type AttributeType = uint
```

Comment:
With the exception of the `CkaFoo` constants, exported types and functions must have comments. (I thought the CI checked this, but obviously not). This comment applies throughout this file (only mentioned once).

Reply:
I've added comments to the various types and functions.